editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3426,6 +3447,41 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	drag = DRAG_NONE;
 	bone_last_frame = 0;
 	additive_selection = false;
+
+	toolbar = memnew(HBoxContainer);
+	toolbar->set_h_size_flags(SIZE_EXPAND_FILL);
+
+	toolbar->set_alignment(BoxContainer::ALIGN_END);
+	hb->add_child(toolbar);
+
+	Label *mouse_pos_txt = memnew(Label);
+	toolbar->add_child(mouse_pos_txt);
+	mouse_pos_txt->set_align(Label::ALIGN_RIGHT);
+	mouse_pos_txt->set_text(TTR("X:"));
+	mouse_pos_txt->add_font_override("font", get_font("source", "EditorFonts"));
```

Comment:
Not sure about this. If the user sets a custom source font in the EditorSettings, this will change, right?
So what's the right way to get a mono-spaced font?